### PR TITLE
Clarify devtools.panels.create() path resolution

### PR DESCRIPTION
**Pull request by bershanskiy**
### Description

Fix mistake in description of `devtools.panels.create()`.

### Motivation

Make documentation correct.

#### Chromium
[Chrome docs state](https://developer.chrome.com/docs/extensions/reference/devtools_panels/#method-create):
> `iconPath`
> Path of the panel's icon relative to the extension directory.
> `pagePath`
> Path of the panel's HTML page relative to the extension directory.

#### Safari
Safari 16 Beta Release Notes list under "Safari Web Extensions > New Features" there is "Added support for Safari Web Inspector Extensions", which most likely includes this method. I did not find documentation about Safari 16 behavior so I tested Safari Technology Preview 153 (future Safari 16.0) and it used paths relative to `manifest.json` (like Chrome). I created [this project for testing](https://github.com/bershanskiy/w3c-webextensions-demo/).

Note: some popular DevTools extensions seem to work around this difference by placing the DevTools background page into the same folder as `manifest.json`.

### Related issues and pull requests
This difference discussed in W3C WebExtensions working group: https://github.com/w3c/webextensions/issues/270

### CODE REVIEW

**Comment by bershanskiy** [comment]
The flaw is not caused by this PR, it existed prior to this PR and was caused by BCD. Also, that underlying problem was already fixed in BCD itself: https://github.com/mdn/browser-compat-data/pull/17755

**Comment by bershanskiy** [comment]
Safari 16 was released, and the BCD PR was merged (linked above), is there anything else I could do to advance this PR?

**Comment by jpmedley** [comment]
@bershanskiy As an MDN reviewer and Chrome's new person on extensions, I'm one of the most logical people to review this. I have no corrections to request. I haven't merged this because non-normative implementation differences are usually handled in BCD notes. I actually think that what you did might be temporarily correct for extensions, but I need to discuss this with Ruth. Can you hang on a bit longer?

**Comment by bershanskiy** [comment]
Thanks for the reply, I just wanted to ensure I'm not blocking anything. Regarding possible MDN update: it was not immediately clear what is the "correct"/"standard" way of handling paths, so I was not sure whether the comment should be on Firefox or all other browsers and whether subfeature should be called "Relative path resolution" or "Absolute path resolution".

**Comment by jpmedley** [comment]
I see that you're the one who opened this ticket in the spec. Are you involved in extensions or did it have more to do with you being a w3c employee? If the former, I think we might be ploying the same field. Can you send me a private email so we can compare notes? jmedley at google dot com.

Were you at TPAC last week?

**Comment by bershanskiy** [comment]
Thanks for the prompt reply! I'm just someone interested in WebExtensions and I'm not affiliated with W3C aside from being a member of a few community working groups. I missed TPAC last week, although I wish I didn't. I'll follow up in an email soon.

**Comment by bsmth** [comment]
Hi @bershanskiy, @jpmedley - just wanted to check in on the status of this one. Are we waiting on a decision whether to have browser-specific details in the prose or not? Thanks :)

**Comment by jpmedley** [comment]
As per [this comment](https://github.com/mdn/content/pull/26908#issuecomment-1584276360), can you please add the implementation differences to [BCD repo](https://github.com/mdn/browser-compat-data)? 

Do you need help with that?

File: files/en-us/mozilla/add-ons/webextensions/api/devtools/panels/create/index.md [hunk 1]
```diff
@@ -26,9 +26,9 @@ let creating = browser.devtools.panels.create(
 - `title`
   - : `string`. The panel's title. This will appear in the row of tabs along the top of the devtools window, and is the main way the user will be able to identify your panel.
 - `iconPath`
-  - : `string`. Specifies an icon which will be shown next to the title. It's provided as a URL to an image file that's been bundled with your extension. The URL is resolved as relative to the current extension page (unless expressed as an absolute URL, e.g. "/icons/panel.png").
+  - : `string`. Specifies an icon which will be shown next to the title. It's provided as a URL to an image file that's been bundled with your extension. Chromium-based browsers and Safari resolve this URL as absolute, while Firefox resolves this URL as relative to the current extension page (unless expressed as an absolute URL, e.g. "/icons/panel.png").
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/api/devtools/panels/create/index.md:29]:
```suggestion
  - : `string`. Specifies an icon which is shown next to the title. It's provided as a URL to an image file that's been bundled with your extension. The URL may be resolved as an absolute URL or relative to the current extension page. See the browser compatibility data for more information.
```

**Review by rebloor — CHANGES_REQUESTED** [review]
I've now moved this information to https://github.com/mdn/browser-compat-data/pull/20617

**Review by rebloor — APPROVED** [review]

**Pull request merged**